### PR TITLE
Add meeting minutes for 15 Dec 2020

### PR DESCRIPTION
**Pull request by SidFaber**
Also include the whitepaper and the presentation used in the meeting

Signed-off-by: Sid Faber <sid.faber@canonical.com>

### CODE REVIEW

File: meetings/2020_12_15/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# ROS 2 Security Working Group Meeting Minutes
+24 Nov 2020
```
Comment (kyrofa) [inline meetings/2020_12_15/README.md]: I don't think this is the right date.

File: meetings/2020_12_15/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Meeting Recording (TBD) | [Meeting Announcement](https://discourse.ros.org/t/ros-security-wg-breakout-meeting-invited-talk-on-privaros/17848)
```
Comment (kyrofa) [inline meetings/2020_12_15/README.md:4]: We're waiting to receive access to the recording so we can upload.

File: meetings/2020_12_15/README.md (hunk 1 above)
Comment (SidFaber) [reply meetings/2020_12_15/README.md]: Updated with deb2fe866d6a1b873866901299fb43f0c0a2b341

**Review by kyrofa — APPROVED** [review]
Let's go ahead and land while we're waiting for the recording, we'll update later.

**Pull request merged**